README.md

```diff
@@ -146,6 +147,19 @@ apply specific provider logic if supported. If not supported, then basic graph i
 * `openstack_lb_pool_v2`
 * `openstack_lb_member_v2`
 
+## FAQ
+
+### Why is my Graph generated empty?
+
+If a Graph is returned as an empty graph it means that we support one of the Providers you are using on your HCL/TFState but we do
+not recognize any connection or relevant Node.
+
+To show the configuration without any Provider logic applied you can use the `--raw` flag that will make the logic ignore the Provider
+logic and print everything that we read. If this works, would be good to try to know why it was empty before so we can take a look
```

Comment:
```suggestion
logic and print everything that we read. If it works, it would be good to try to know why it was empty before so we can take a look
```